pages/in/widget/pay/index.vue

```diff
@@ -701,7 +714,7 @@ export default {
       }
     },
     async postTransaction({ txHash, error } = {}) {
-      if (this.redirectUri) {
+      if (this.opener || this.redirectUri) {
```

Comment:
What happen in this part if `this.redirectUri` is not defined?

Reply:
True, then `const url = new URL(this.redirectUri, true);` would have error. Need to change this.